src/authorization/mod.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under the MIT license <LICENSE-MIT
+// https://opensource.org/licenses/MIT> or the Modified BSD license <LICENSE-BSD
+// https://opensource.org/licenses/BSD-3-Clause>, at your option. This file may not be copied,
+// modified, or distributed except according to those terms. Please review the Licences for the
+// specific language governing permissions and limitations relating to use of the SAFE Network
+// Software.
+
+pub mod access_control;
+mod tests;
```

Comment:
We like to keep separation between `pub mod` and `mod` statements because one exports an API publicly and one doesn't. It's useful to keep them in separate blocks so it's easier to see the public exports at a glance.

Reply:
You mean, in this case separation with a line?

Reply:
Exactly! [Example](https://github.com/maidsafe/safe-nd/blob/master/src/lib.rs)